docs/cloud/dev/spryker-cloud-commerce-os/configure-deployment-pipelines/configuring-github-actions.md

```diff
@@ -8,8 +9,844 @@ redirect_from:
+* To learn about encrypted secrets in a workflow on demand, see [Encrypted secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+* To learn about the Deploy file, see [Deploy file reference - 1.0](/docs/scos/dev/the-docker-sdk/{{site.version}}/deploy-file/deploy-file-reference-1.0.html)
+* To learn about configuring services with the Docker SDK, see [Configuring services](/docs/scos/dev/the-docker-sdk/{{site.version}}/configure-services.html)
+</details>
+
+<details>
+<summary>Release 202212.0</summary>
+This document describes how to configure continuous integration using GitHub Actions.
+
+## GitHub Actions
+
+GitHub Actions helps you automate your software development workflows in the same place you store code and collaborate on pull requests and issues. You can write individual tasks, called actions, and combine them to create a custom workflow. Workflows are custom automated processes that you can set up in your repository to build, test, package, release, or deploy any code project on GitHub.
+
+For more information on GitHub Actions, see the following documents:
+* [GitHub Actions](https://github.com/features/actions)
+* [Learn GitHub Actions](https://docs.github.com/en/actions/learn-github-actions)
+
+
+## Prerequisites
+
+In a GitHub repository, create the workflow configuration file: `.github/workflows/{project_name}.yml`.
+
+## Configuring basic validation with GitHub Actions
+
+Set up the job that performs the basic validation like the following:
+* Code style
+* Architecture
+* Security
+* Database schema
+
+
+<details open>
+<summary>Example of .github/workflows/{project_name}.yml</summary>
+
+```yaml
+...
+jobs:
+    ...
+    validation:
+        name: "CS, ArchSniffer, PHPStan, Security"
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version: [
+                    '7.4'
+                ]
+
+        env:
+            APPLICATION_ENV: devtest
+            APPLICATION_STORE: DE
+            PROJECT: project-name
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Configure sysctl limits
+              run: |
+                  sudo swapoff -a
+                  sudo sysctl -w vm.swappiness=1
+                  sudo sysctl -w fs.file-max=262144
+                  sudo sysctl -w vm.max_map_count=262144
+            - name: Runs Elasticsearch
+              uses: elastic/elastic-github-actions/elasticsearch@master
+              with:
+                  stack-version: 7.6.0
+                  port: 10005
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: '16'
+
+            - name: NPM cache
+              uses: actions/cache@v2
+              with:
```

Comment:
The same.

Reply:
Updated according to [PR](https://github.com/spryker-shop/suite/pull/1240)
Other changes have to be changed by DevOps or BE dev